CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing guidelines
+
+### Contributing code
+If you have improvements to TAU, send us your pull requests! For those just getting started, Github has a [howto](https://help.github.com/articles/using-pull-requests/).
+
+### Commit policy
+We recommend that use a 1 commit for the pull request.
+
+### Commit message
```

Comment:
When viewed ([1]) lines in commit message are not broken. Line size restrictions?

[1] https://github.com/Samsung/TAU/blob/8c4efcd7cb0d59022319f0eafe1f621f60dce96e/CONTRIBUTING.md